src/locale/locales/de/messages.po

```diff
@@ -636,32 +602,28 @@ msgstr "Überprüfe deinen Posteingang auf eine E-Mail mit dem Bestätigungscode
 
 #: src/view/com/modals/Threadgate.tsx:72
 msgid "Choose \"Everybody\" or \"Nobody\""
-msgstr ""
+msgstr "Wählen Sie \"Alle\" oder \"Niemand\""
 
 #: src/view/screens/Settings/index.tsx:697
 msgid "Choose a new Bluesky username or create"
-msgstr ""
+msgstr "Wähle einen neuen Bluesky-Benutzernamen oder erstelle"
 
 #: src/view/com/auth/server-input/index.tsx:79
 msgid "Choose Service"
 msgstr "Service wählen"
 
 #: src/screens/Onboarding/StepFinished.tsx:135
 msgid "Choose the algorithms that power your custom feeds."
-msgstr ""
+msgstr "Wählen Sie die Algorithmen, die Ihre benutzerdefinierten Feeds betreiben."
 
 #: src/view/com/auth/onboarding/WelcomeDesktop.tsx:83
 #: src/view/com/auth/onboarding/WelcomeMobile.tsx:83
 msgid "Choose the algorithms that power your experience with custom feeds."
-msgstr ""
-
-#: src/screens/Onboarding/StepAlgoFeeds/index.tsx:103
-#~ msgid "Choose your algorithmic feeds"
-#~ msgstr ""
+msgstr "Wählen Sie die Algorithmen, die Ihr Erlebnis mit benutzerdefinierten Feeds unterstützen."
 
 #: src/screens/Onboarding/StepAlgoFeeds/index.tsx:103
 msgid "Choose your main feeds"
-msgstr ""
+msgstr "Wählen Sie Ihre wichtigsten Feeds"
```

Comment:
"wichtigsten" means "most important". I guess you could equal that to "main".

Reply:
Would there be a more direct translation that would work better? If I translate "main" on its own DeepL gives me "Haupt".

Reply:
And that is correct. I'm unsure, though. It would then read

```suggestion
msgstr "Wählen Sie Ihre Haupt-Feeds"
```

I guess you can leave that as is.

Reply:
OK I'll leave this one open for a bit more feedback

Reply:
both would work but since Haupt-Feeds is closer to the original I'd go with this one